internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -53,6 +64,79 @@ func (s *Server) SetUpSession(stream s2av2pb.S2AService_SetUpSessionServer) erro
 	}
 }
 
+// TODO(rmehta19): consider also returning an error with this function.
+func validatePeerCertificateChain(req *s2av2pb.ValidatePeerCertificateChainReq) (*s2av2pb.SessionResp, error) {
+	switch x := req.PeerOneof.(type) {
+	case *s2av2pb.ValidatePeerCertificateChainReq_ClientPeer_:
+		// TODO(rmehta19): Handle Verification Mode(UNSPECIFIED, SPIFFE, or CONNECT_TO_GOOGLE).
+		derCertChain := req.GetClientPeer().CertificateChain
+		if len(derCertChain) == 0 {
+			// TODO(rmehta19): Is it an error for peer to have empty cert chain(I think
```

Comment:
Is an empty cert chain in the SessionReq an error that S2A will raise?

Reply:
Yup!